transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -58,7 +58,7 @@ public Socket(int fd) {
     /**
      * Returns {@code true} if we should use IPv6 internally, {@code false} otherwise.
      */
-    private boolean useIpv6(InetAddress address) {
+    protected boolean useIpv6(InetAddress address) {
```

Comment:
nit: should we rename this to `isIPv6()` ?

Reply:
there's already a method by that name